example-playbooks/replace_node/replace_node.yml

```diff
@@ -72,3 +72,129 @@
             io_properties: "{{ _io_properties.stdout }}"
       delegate_to: "{{ alive_node }}"
       run_once: true
+
+
+# This play will install Scylla in the new node and update the existing nodes.
+# The same variables used when the node role was executed for the replaced node should also
+# be passed to this playbook and will be used by this play.
+- name: Install/Update Scylla in all nodes
+  hosts: scylla
+  vars:
+    firewall_enabled: true
+  roles:
+    - ansible-scylla-node
```

Comment:
Don't you want to set `don't start scylla service` somewhere? 
If you did, where is it?

Reply:
It's set in `vars/main.yml`